web/src/settings_invites.ts

```diff
@@ -110,6 +127,9 @@ function populate_invites(invites_data: {invites: Invite[]}): void {
             item.img_src = people.small_avatar_url_for_person(
                 people.get_by_user_id(item.invited_by_user_id),
             );
+            if (!settings_data.user_can_create_multiuse_invite()) {
+                item.disable_buttons = true;
```

Comment:
I think this will disable all revoke and resend buttons as well, even for email invites, when user is not allowed to create multiuse invites, which we don't want.

Reply:
But then also, all the buttons will be disabled, right? We only need to disable edit button.

Reply:
It is added again for all the buttons while rebasing. Also make sure that we do not disable other buttons in any case and it will be fine to remove it for other buttons.